Lidar/lidarPositioning.py

```diff
@@ -0,0 +1,139 @@
+from rplidar import RPLidar
+import time
+import math
+
+
+
+class fieldScanner:
+    """A frame manager for lidar. This class is responsible for reading lidar data 
+    and maintainning a data frame the represents the latest 360 degree view of the field"""
+
+    mLidar = None
+   
+    def __init__(self, lidarDevice):
+            self.mLidar = lidarDevice
+
+    def getCurrentPosition(self,gyro):
```

Comment:
you might want to consider renaming & refactoring this method to something like getFieldScan() since we will now do navigation in a separate class